src/Symfony/Component/OpenApi/Dumper/DumperInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\OpenApi\Dumper;
+
+use Symfony\Component\OpenApi\Model\OpenApi;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ * @author Selency Team <tech@selency.fr>
+ */
+interface DumperInterface
+{
+    public function dump(OpenApi $compiledDoc): string;
```

Comment:
I'm not sure this DumperInterface is useful. I doubt we will have code that can work with _any_ dumper without knowing ore about them (and we might have cases where we would want to dump a compiled doc to a static website for instance, which would not fit with returning a string)